deblur/workflow.py

```diff
@@ -7,20 +7,39 @@
 # -----------------------------------------------------------------------------
 
 
-def trim_seqs(seqs_fp, trim_len):
-    """Step 1: trim FASTA sequences to specified length"""
-    pass
+from os.path import splitext
+from bfillings.vsearch import vsearch_dereplicate_exact_seqs
 
 
-def dereplicate_seqs(seqs_fp):
-    """Step 2a: dereplicate FASTA sequences using VSEARCH"""
+def trim_seqs(seqs_fp, trim_len):
+    """Step 1: trim FASTA sequences to specified length"""
     pass
 
 
-def remove_singletons_seqs(seqs_fp):
-    """Step 2b: remove singletons from dereplicated FASTA
-       file using VSEARCH"""
-    pass
+def dereplicate_seqs(seqs_fp,
+                     output_fp,
+                     minuniquesize=2):
+    """Step 2a: dereplicate FASTA sequences and remove
+       singletons using VSEARCH
+
+    Parameters
```

Comment:
change `minuniquesize` to `min_size` (the name of the variable is complex and not even PEP8 - although it doesn't have a way of flagging it) and add to the documentation.